Basic-Video-Chat-With-ForegroundServices-Java/app/build.gradle

```diff
@@ -0,0 +1,44 @@
+plugins {
+    id 'com.android.application'
+}
+
+apply {
+    from '../../commons.gradle'
+}
+
+android {
+    compileSdkVersion extCompileSdkVersion
+
+    defaultConfig {
+        applicationId "com.tokbox.sample.basicvideochatwithforegroundservices"
+        minSdkVersion extMinSdkVersion
+        targetSdkVersion extTargetSdkVersion
+        versionCode extVersionCode
+        versionName extVersionName
+    }
```

Comment:
Wouldn't we like to target something more recent? I see https://github.com/opentok/opentok-android-sdk-samples/blob/VIDCS-3351/commons.gradle#L8-L11 and seems old.

Reply:
yes, 34, just like our SDK